packages/store-sync/src/recs/syncToRecs.ts

```diff
@@ -59,9 +64,48 @@ export async function syncToRecs<TConfig extends StoreConfig = StoreConfig>({
 
   world.registerDisposer(stopSync);
 
+  const functionSelectorToResourceSelector = new Map<Hex, Hex>();
+
+  const getResourceSelector = async (functionSelector: Hex): Promise<Hex> => {
+    if (functionSelectorToResourceSelector.has(functionSelector)) {
+      return functionSelectorToResourceSelector.get(functionSelector) as Hex;
+    }
+
+    const entity = encodeEntity<KeySchema>(components.FunctionSelectors.metadata.keySchema, {
+      functionSelector,
+    });
+
+    const selectors = getComponentValue(components.FunctionSelectors, entity);
```

Comment:
this only returns
```suggestion
    const selector = getComponentValue(components.FunctionSelectors, entity);
```
this only returns one selector

Reply:
Looks like it returns both `resourceSelector` and `systemFunctionSelector`...? We are only grabbing `resourceSelector` here for sure

Reply:
you're right, sorry! didn't realize we had two properties on this table